draft-ietf-hpke-pq.md

```diff
@@ -188,7 +188,12 @@ The constants `Nsecret` and `Nsk` are always 32 and 64, respectively.  The
 constants `Nenc` and `Npk` depend on the ML-KEM parameter set in use; they are
 specified in {{ml-kem-iana-table}}.
 
-# Hybrid KEMs with ECDH and ML-KEM
+{:aside}
+> Note: While this document defines an HPKE KEM for ML-KEM-512 in the interest
+> of completeness, the security level that ML-KEM-512 provides is not generally
```

Comment:
It's a bit more subtle than that. At the time of writing there is no peer-reviewed evidence that ML-KEM-512 doesn't reach its claimed security level (AES-128). ML-KEM-768 is a hedge against cryptanalytic improvement.